ProcessesApi/HttpHeadersExtensions.cs

```diff
@@ -5,7 +5,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace BaseApi
+namespace ProcessesApi
```

Comment:
This file can be deleted and you can use to the Middleware nuget package instead